hbase_sql/src/main/java/com/splicemachine/derby/stream/spark/SparkDataSet.java

```diff
@@ -300,6 +303,20 @@ public long count() {
         return union(dataSet, RDDName.UNION.displayName(), false, null);
     }
 
+
+    @Override
+    public DataSet<V> parallelProbe(List<DataSet<V>> dataSets) {
+        DataSet<V> toReturn = null;
+        int i = 0;
+        for (DataSet<V> dataSet: dataSets) {
```

Comment:
This function is not used, but the logic is wrong here, since we are not incrementing 'i'.